src/api/app/views/webui2/webui/cloud/ec2/configurations/show.html.haml

```diff
@@ -0,0 +1,53 @@
+.card.mb-3
+  .card-body
+    %h3.mb-3
+      Amazon EC2 Configuration
+    %p
+      Amazon EC2 allows you to easily run instances of your appliance on Amazon's webservers.
+
+    %h4
+      Roles and permissions
+
+    %p
+      To upload your appliances to Amazon EC2, the Open Build Service requires permission to run EC2 instances. This may incur costs,
+      %a{ href: 'https://aws.amazon.com/partners/suse/' }
```

Comment:
I would add a target `_blank` on this link since it's an external link.

Reply:
Added.